XRTK-Core/Packages/com.xrtk.core/Services/MixedRealityToolkit.cs

```diff
@@ -1013,7 +1085,7 @@ private static bool RegisterServiceInternal(Type interfaceType, IMixedRealitySer
         /// </summary>
         /// <param name="interfaceType">The interface type for the system to be removed.  E.G. InputSystem, BoundarySystem</param>
         /// <param name="serviceName">The name of the service to be removed. (Only for runtime services) </param>
-        private static bool UnregisterServiceInternal(Type interfaceType, string serviceName)
+        private static bool TryUnregisterServiceInternal(Type interfaceType, string serviceName)
```

Comment:
If we now have service parenting, if I try to unregister a parent service, it should try and unparent all it's children first (any service who has a parent the same as this service.

Reply:
Great suggestion!

I'll be sure to add that as well

Reply:
Added!